README.md

```diff
@@ -1 +1,15 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## Biography
+
+![Profile Image](/profile.jpg)
+* **Name:** Hanyang (Henry) Zhang
+* **Program:** MAPSS Econ
+* **Hometown:** Hangzhou
+* **Research Interest:**
+  * Applied Microeconomic Theory
+  * Game Theory
+  * Applied Microeconomics
+* **Email:** <zhanghanyang@uchicago.edu>
```

Comment:
Great job linking your email through a URL.

Reply:
Hi Neeraj! Thanks for your comments. I wonder if I can get feedbacks on why some of the criteria of the evaluation are satisfactory, and how I can possibly improve them, so that I might do better next time? Thanks!